contributions/catalog/models/capability_utilities.py

```diff
@@ -34,8 +34,9 @@ def init_capability():
              'environmentVariables': []
          },
          'dataDeletionEndpointDetails': {
-             'endpoint': '',
-             'api': ''
+             'deletionEndpoint': '',
+             'apiKey': '',
+             'description': ''
```

Comment:
I do not see a description field in the 'Data Deletion' form section for capability. Is this required?

Reply:
That is also my question. Maybe we need to discuss about this. There are some data discrepancy between the yaml and the form.

Reply:
I think the description field was probably included to store any remarks about the delete endpoint. We can revisit this.